doc/tutorials/platformer/step_1.md

```diff
@@ -0,0 +1,99 @@
+# 1. Preparation
+
+Before you begin any kind of game project, you need an idea of what you want to make and I like to
+then give it a **name**. For this tutorial and game, I got inspiration from @spydon's tweet about
+the Flame game engine GitHub repository reaching 7000 stars and I thought, excellent! Ember will be
+on a quest to gather as many stars as possible and I will call the game, `Ember Quest`.
+
+![Tweet](../../images/tutorials/platformer/tweet.jpg)
```

Comment:
Haha I appreciate the shout-out, but I think this could be simplified to something like this: 
```suggestion
Before you begin any kind of game project, you need an idea of what you want to make and I like to
then give it a **name**. For this tutorial and game, Ember will be on a quest to gather as many
(GitHub) stars as possible and I will call the game, `Ember Quest`.
```